.cmake-format

```diff
@@ -26,7 +26,8 @@ with section("parse"):
       'kwargs': {
         'NAME': '*', 
         'SRCS': '*', 
-        'LIBS': '*'}},
+        'LIBS': '*',
+        'CFGS': '*',}},
```

Comment:
Redundant `,`. It should be `'CFGS': '*'}},`

Reply:
Removed